docs/cross-references.md

```diff
@@ -371,3 +371,5 @@ Finally, under the `numbering` object, you may specify `enumerator`. For now, th
 numbering:
   enumerator: A1.%s
 ```
+
+If you want to control the numbering for a specific figure, you can use the {myst:directive}`figure.enumerator` option. This will give the figure a specific enumerator and not increment the counting for other figures. This is helpful if you want to explicitly count figure `2a` and then carry on counting figures as normal; alternatively you can take control of numbering entirely.
```

Comment:
```suggestion
If you want to control the numbering for a specific figure, you can use the {myst:directive}`figure.enumerator` option. This will give the figure a specific enumerator, and will not increment the counting for other figures. This is helpful if you want to explicitly count figure `2a` and then carry on counting figures as normal; alternatively you can take control of numbering entirely.
```

@rowanc1 can you elaborate on 
> alternatively you can take control of numbering entirely.

Is this saying 
> you can just set `enumerator` on every enumerable?

Reply:
Yes!